src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(x) {
+      return this.result + x;
+    },
+
+    subtract(x) {
+      return this.result - x;
+    },
+
+    multiply(x) {
+      return this.result * x;
+    },
+
+    divide(x) {
+      if (x === 0) {
+        throw new Error('Division by zero is not allowed.');
+      }
+
+      return this.result / x;
```

Comment:
The `divide` method should update the `result` property of the calculator object. Consider changing the implementation to `this.result /= x;` to ensure the calculator's state is updated.